examples/plot_credit_loan_decisions.py

```diff
@@ -652,12 +648,13 @@ def plot_group_metrics_with_error_bars(metricframe, metric, error_name):
 # Postprocessing mitigations: ThresholdOptimizer
 # ==============================================
 # In the Fairlearn package, *postprocessing* mitigation is offered through
-# the :code:`ThresholdOptimizer` algorithm, following `Hardt, Price, and Srebro (2016)
-# <https://arxiv.org/abs/1610.02413>`_. The :code:`ThresholdOptimizer` takes in an
-# exisiting (possilbly pre-fit) machine learning model whose predictions acts as
-# a scoring function to identify separate thresholds for each *sensitive
-# feature* group. The :code:`ThresholdOptimizer` optimizes a specified objective
-# metric (in our case, :code:`balanced_accuracy`) subject to some fairness constraint
+# the :code:`ThresholdOptimizer` algorithm, following Hardt, Price, and Srebro
+# (2016) :footcite:`hardt2016equality`.
```

Comment:
Again, use one of the other `:footcite:` modes to generate the author list and year automatically.